lib/Doctrine/ORM/Sequencing/SequenceGenerator.php

```diff
@@ -70,41 +64,31 @@ public function generate(EntityManagerInterface $em, $entity)
 
     /**
      * Gets the maximum value of the currently allocated bag of values.
-     *
-     * @return int|null
      */
-    public function getCurrentMaxValue()
+    public function getCurrentMaxValue() : ?int
     {
         return $this->maxValue;
     }
 
     /**
      * Gets the next value that will be returned by generate().
-     *
-     * @return int
```

Comment:
Was this previously wrong?

Reply:
yeap...

Reply:
I mean, it's is not needed with the type hint =)

Reply:
The hint added nullability

Reply:
Hmm, no, I think my comment was misplaced